commands.c

```diff
@@ -1068,10 +1068,12 @@ int mutt_check_traditional_pgp (HEADER *h, int *redraw)
   else
   {
     for (i = 0; i < Context->vcount; i++)
-      if (Context->hdrs[Context->v2r[i]]->tagged && 
-	  !(Context->hdrs[Context->v2r[i]]->security & PGP_TRADITIONAL_CHECKED))
-	rv = _mutt_check_traditional_pgp (Context->hdrs[Context->v2r[i]], redraw)
-	  || rv;
+      if ((Context->hdrs[Context->v2r[i]]->tagged) && 
+           (Context->hdrs[Context->v2r[i]]->security & PGP_TRADITIONAL_CHECKED))
+      {
+        rv = _mutt_check_traditional_pgp (Context->hdrs[Context->v2r[i]],redraw)
+                  || rv;
+      }
   }
```

Comment:
why did you do the change above? it's unrelated to the other parts of that commit. I think it should be either another commit or another PR for cosmetics only.